Examples/test-suite/fortran_bindc_all.i

```diff
@@ -0,0 +1,31 @@
+%module fortran_bindc_all
+
+%{
+#include <stdlib.h>
+%}
+
+// Give *all* types the struct typemap by default
+%fortran_bindc_struct(SWIGTYPE);
+// Apply the "bindc" feature to everything
+%bindc "1";
+
+%feature("matched_name") DBmrgtree   "! OK     DBmrgtree   ";
+%feature("matched_name") f_DBmrgtree "ERROR  ! f_DBmrgtree ";
+%feature("matched_name") _DBmrgtree  "ERROR  ! _DBmrgtree  ";
```

Comment:
I can't find any implementation for the matched_enum feature.

Reply:
Yep that was inadvertently left in there while debugging the name matching properties of `%feature` for `typedef struct _foo { /* * */ } foo;` . I have removed it.